Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.8-slim
+WORKDIR /mote
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+RUN sed -i 's|json_cache_location = "/var/cache/httpd/mote/cache.json"|json_cache_location = "./cache.json"|' ./mote/config.py
+EXPOSE 5000
+COPY . .
+CMD ["python3", "runmote.py"]
```

Comment:
I would suggest, removing it from here, as It'll hardcode the `runmote.py` in the container image, and for some cases, a developer might have to run another file.

We can solve it by adding it to `docker-compose.yml`

Reply:
Makes sense.

Also, L#8 does not look very elegant to me.

Reply:
I could use some assistance regarding the reference file locations :sweat_smile: